SAPHana/bin/SAPHanaSR-manageAttr

```diff
@@ -883,11 +889,17 @@ chk_for_maintenance() {
     # check maintenance mode of ms and cln resource
     # Attention: if the keyword in the cib will change from 'master' to 'multistate'
     # or something else, we need to adapt the following line
-    resM="$(get_res master)"
+    resM="$(get_res master)"; answer=$?
```

Comment:
Could you explain the usage of ino_match vs INO_match? Is one a local and the other global?

Reply:
INO_match is a global variable. But as 'get_res' is called in a subshell, it's not possible to change INO_match directly in this function. So I use a local variable ino_match inside the function and use the return value of the function to have the information available outside the function without violating the 'output' of the function, which is needed to set the resource name.

Reply:
+1